bpf/bpf_lxc.c

```diff
@@ -1042,10 +1042,12 @@ __section_tail(CILIUM_MAP_POLICY, LXC_ID) int handle_policy(struct __sk_buff *sk
 		return send_drop_notify(skb, src_label, SECLABEL, LXC_ID,
 					ifindex, ret, TC_ACT_SHOT);
 
-	ifindex = skb->cb[CB_IFINDEX];
+	if ifindex == skb->cb[CB_IFINDEX] { // Not redirected to host / proxy.
```

Comment:
This is fine but needs `(` `)`

Reply:
Doh. I did run make, but missed the error. Fixed.